src/tests/unit/tests/background/actions/action-creator.test.ts

```diff
@@ -1164,11 +1171,16 @@ class ActionCreatorValidator {
         actionName: string,
         actionsMap: DictionaryStringTo<IMock<Action<any, any>>>,
         actionsContainerMock: IMock<any>,
+        asyncAction: boolean = false,
     ): ActionCreatorValidator {
         let action = actionsMap[actionName];
 
         if (action == null) {
-            action = Mock.ofType(SyncAction);
+            if (asyncAction) {
+                action = Mock.ofType(AsyncAction);
+            } else {
+                action = Mock.ofType(SyncAction);
+            }
```

Comment:
Could SyncAction or AsyncAction be passed as a parameter or a generic type instead of the boolean, so this could be simplified as `action = Mock.ofType<ActionType>()`?